.github/workflows/testing.yaml

```
@@ -20,7 +20,11 @@ jobs:
         tarantool:
           - 'debug-master'
           - '3.1'
-
+        include:
+          - tarantool: 'debug-master'
+            coveralls: true
+          - tarantool: '3.1'
+            coveralls: false
```

```suggestion
        tarantool:
           - 'debug-master'
        include:
          - tarantool: '3.1'
            coveralls: true
```
`debug-master` could be slow to build and could cause errors in case of loss of backward compatibility.

So I suggest just run regular tests with `debug-master` and with coverage for a latest stable version.